sound/hybrid_font.h

```diff
@@ -231,13 +240,24 @@ class HybridFont : public SaberBase {
         }
         if (!swing_player_) {
           if (!swinging_) {
-            if (rss > slashThreshold && SFX_slsh) {
-              swing_player_ = PlayPolyphonic(&SFX_slsh);
-            } else if (SFX_swng) {
-              swing_player_ = PlayPolyphonic(&SFX_swng);
+            if (font_config.ProffieOSMaxSwingAcceleration > font_config.ProffieOSMinSwingAcceleration) {
+              float s = (rss - font_config.ProffieOSMinSwingAcceleration) / font_config.ProffieOSMaxSwingAcceleration;
+              if (SFX_swng) {
```

Comment:
I think we should reorganize the code a little so that it first picks the right Effect, then assigns that to an Effect* variable. Then we'll do the if statement that decides if we're doing a SelectFloat() based on the rss, and then we'll just call PlayPolyphonic. Should simplify the code a bit.

Reply:
Updated.

Reply:
I think we should use exactly the same logic for selecting the effect regardless of whether ProffieOSMaxSwingAceleration is set or not.  It may lead to some weird things if you try to use swings and slashes at the same time, but it might also be useful...

Reply:
My thinking is that nobody should have both accent slashes and swing selection configured at the same time, so we might as well make the code as simple as possible with the assumption that only one of those will be active.

Reply:
Sorry, maybe I'm missing something I have it so that if ProffieOSMaxSwingAcceleration is set (> Min) slsh isn't used at all but if it 's not then slsh is via the "else" but only if it exists.  Is there a better way to do this, it's escaping me at present?

Reply:
Nevermind, I think I understand was looking at it backwards? Is this what you meant?